lib/src/search/match.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:equatable/equatable.dart';
+
+class SearchMatch extends Equatable {
```

Comment:
Try to use the standard `Match`. See how it compares instances.

Reply:
`Match` is an abstract class. But we want to be able to manually create instances of it.
It also contains a lot of properties that are not needed for us.

Seems like it is better to leave SearchMatch as is.